index.js

```diff
@@ -88,6 +88,8 @@ app.get('/recommend', (req, res) => {
     case 'vegetables':
       recommendation = 'Carrot';
       break;
+    case 'keyboards':
+      recommendation = 'Logitech MX Keys';
```

Comment:
```suggestion
      recommendation = 'Logitech MX Keys';
      break;
```
Adding the break makes sure it does not go to the default case (Unknown product type)!